meta/execution-environment.yml

```diff
@@ -0,0 +1,15 @@
+---
+# Copyright (c) Ansible Project
+# GNU General Public License v3.0+ (see LICENSES/GPL-3.0-or-later.txt or https://www.gnu.org/licenses/gpl-3.0.txt)
+# SPDX-License-Identifier: GPL-3.0-or-later
+
+# List your controller-side system package and/or Python package requirements
+# if exist in the files specified below to support easy execution environments building for your collection users
+# https://ansible.readthedocs.io/en/latest/getting_started_ee/index.html
+# See https://ansible.readthedocs.io/projects/builder/en/latest/collection_metadata/#how-to-verify-collection-level-metadata
+# to learn how to test your configuration.
+# Do not delete these files even if the collection has no dependencies!
```

Comment:
How about explaning this a bit more here:
```suggestion

# Do not delete these files even if the collection has no dependencies!
# If you delete them, ansible-builder will use bindep.txt and requirements.txt
# from the collection root by default. Especially requirements.txt there is often
# used to test/build/development requirements for the collection, that are
# not needed for end-users to use the collection. This would increase the EE
# size and increase the chance of dependency collisions.
```

Reply:
@felixfontein Good warning, committed, thanks, PTAL